test/conformance/testdata/conformance.yaml

```diff
@@ -1,2909 +1,1611 @@
 - testname: Pod Lifecycle, post start exec hook
```

Comment:
this delta will go away once yaml.v2 v2.4.0 is picked up

Reply:
ack